components/consumers/pdf/Makefile

```diff
@@ -0,0 +1,18 @@
+.PHONY: container publish
+
+CONTAINER_REPO=
+DRACON_VERSION=
+SOURCE_CODE_REPO=
+PRODUCER_AGGREGATOR_BASE_IMAGE=$(shell test -e .custom_image && cat .custom_image || echo "scratch")
+
+DOCKER=docker
+
+container:
+	cp default.html ../../../bin/components/consumers/pdf/default.html
+	$(DOCKER) build --tag $(CONTAINER_REPO)/components/consumers/pdf:$(DRACON_VERSION) \
+					--file Dockerfile \
+					$$([ "${SOURCE_CODE_REPO}" != "" ] && echo "--label=org.opencontainers.image.source=${SOURCE_CODE_REPO}" ) \
+					 ../../../bin 1>&2
+
+publish:
+	$(DOCKER) push $(CONTAINER_REPO)/components/consumers/pdf:$(DRACON_VERSION) 1>&2
```

Comment:
this publish doesn't look like it has a different tag than the one generated automatically

Reply:
no it doesn't, i just copied the Makefile from other consumers

Reply:
so it should be removed if its not required?